chit_chat/migrations/0001_initial.py

```diff
@@ -30,17 +29,22 @@ class Migration(migrations.Migration):
                 ('archived', models.BooleanField(default=False)),
                 ('ignore_notifications', models.BooleanField(default=False)),
                 ('created_when', models.DateTimeField(default=django.utils.timezone.now)),
-                ('room', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='memberships', to='chit_chat.Room')),
+                ('room', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='memberships', to='chit_chat.room')),
                 ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='chat_room_memberships', to=settings.AUTH_USER_MODEL)),
             ],
         ),
+        migrations.AddField(
```

Comment:
Only concern here is compatibility with applications already running chit-chat.

What could make sense is:
1. Create new m2m ,`members_tmp`, with though model.
2. Create data migration to load new field with old field's data.
3. Delete old m2m field.
4. Rename new m2m `members_tmp` => `members`.

Reply:
pushed this up, hand tested a couple times, should worky!